addons/web/static/src/views/relational_model/record.js

```diff
@@ -57,6 +57,19 @@ export class Record extends DataPoint {
         return this.isNew && !this.isDirty && !this._canNeverBeAbandoned;
     }
 
+    get evalContext() {
+        return Object.assign(
+            {
+                active_id: this.resId || false,
+                active_ids: this.resId ? [this.resId] : [],
+                active_model: this.resModel,
+                current_company_id: this.model.company.currentCompany.id,
+            },
+            this.context,
+            this.dataContext
+        );
```

Comment:
it's not equivalent to before (e.g. active_id key in context would override the active_id defined here)
```suggestion
    get evalContext() {
        return Object.assign(
            {},
            this.context,
            {
                active_id: this.resId || false,
                active_ids: this.resId ? [this.resId] : [],
                active_model: this.resModel,
                current_company_id: this.model.company.currentCompany.id,
            },
            this.dataContext
        );
```